lib/criticalcss.js

```diff
@@ -74,7 +74,7 @@ criticalCSS by @scottjehl && @jefflembeck. Run this on your CSS, get the styles
 		critical = critical.filter( noEmpties );
 
 		if( critical.length  === 0 ){ return ""; }
-		return critical.join( "\n" );
+		return critical;
```

Comment:
This particular change should be scrutinized. I've made the assumption that dependencies on this result are confined to the library but if there are other clients this file then this could be an issue.